RELEASE_NOTES.md

```diff
@@ -20,6 +20,9 @@
 
 - The `Fuse` class has been moved to the `frequenz.sdk.timeseries` module.
 
+- The `ComponentGraph.components()` parameters `component_id` and `category_id`
+  were renamed to `component_ids` and `category_ids`, respectively.
```

Comment:
Wrong name `category_id`(s) and also GitHub release notes doesn't like line breaks.

```suggestion
- The `ComponentGraph.components()` parameters `component_id` and `component_categories`   were renamed to `component_ids` and `component_categories`, respectively.
```